apps/gitness/app_constants.ts

```diff
@@ -0,0 +1,3 @@
+export const UI_DEV_SERVER_PORT = 5137 // UI dev server port
```

Comment:
this file is overkill for now.. we dont need it.
instead, lets set up `dotenv` so that we can configure these things via env variables

Reply:
Done